app/src/main/res/layout/activity_main_screen.xml

```diff
@@ -121,4 +121,17 @@
         app:layout_constraintTop_toBottomOf="@+id/house_imageButton"
         app:layout_constraintVertical_bias="0.064" />
 
+    <Button
+        android:id="@+id/add_household_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="100dp"
+        android:layout_marginEnd="56dp"
+        android:onClick="addHouseholdButtonPressed"
+        android:text="@string/add_household"
+        app:layout_constraintEnd_toStartOf="@+id/refresh_calendar"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
```

Comment:
The button is displayed a little on top of the calendar.
It is temporary I know but not pretty for the presentation ^^
![ecranAddHouse](https://user-images.githubusercontent.com/34856793/159134254-16db9866-fb6b-44e9-befb-7ccabe2b647c.PNG)

Reply:
I know it was just a placeholder for now. I will change it when merged